### PR TITLE
Add tags about process and prefix to sentry errors

### PR DESCRIPTION
**Pull request by imbstack**
This might help a bit with debugging queue I hope?

### CODE REVIEW

File: src/monitor.js [hunk 1]
```diff
@@ -30,7 +30,13 @@ class Monitor {
     }).catch(err => {});
 
     this._sentry.then(sentry => {
-      sentry.client.captureException(err, {level});
+      sentry.client.captureException(err, {
+        tags: _.defaults({
+          prefix: this._opts.project + (this._opts.prefix || '.root'),
```
Comment (jonasfj) [inline src/monitor.js:39]: Any need for .root?

File: src/monitor.js [hunk 2]
```diff
@@ -30,7 +30,13 @@ class Monitor {
...
+          process: this._opts.process || 'unknown',
```
Comment (jonasfj) [inline src/monitor.js:40]: Nice

File: src/monitor.js [hunk 3]
```diff
@@ -17,7 +17,7 @@ class Monitor {
     this._resourceInterval = null;
   }
 
-  async reportError(err, level='error') {
+  async reportError(err, level='error', tags={}) {
```
Comment (jonasfj) [inline src/monitor.js:20]: So let's change the behavior here...

IMO let's make two methods reportError(errOrText, tags) and reportWarning(errOrText, tags)

Note, that if tags for reportError is the string 'warning' we can handle that as a special case for backwards compatibility... and eventually remove support for that...

How does that sound... I know if error and warning levels, if there are more we want to support... maybe it's better to have only reportError(errOrText, level, tags) but then if level is an object set tags= level and level=error..
So you don't have to supply a level argument for error which is the common case...

Comment (imbstack) [inline src/monitor.js:20]: I honestly don't expect people to set tags very often at all. I like the second option you mentioned more out of the two if we do one at all.

Comment (jonasfj) [inline src/monitor.js:20]: @imbstack I might very want to set tags every time...

Like:
`monitor.reportError(err, 'warning', {location: 'deadline-reaper-poll-loop'});`
As a way for me to attach a tiny bit of additional information that can tell where this error came from... or:
`monitor.reportError(err, {note: 'this happened somewhere in api.js where, not causing issues but something is very wrong as an internal invariant is broken... search for this message in the code...'});`

Comment (jonasfj) [inline src/monitor.js:20]: I know both of these are hacks :)

Comment (jonasfj) [inline src/monitor.js:20]: note: option 2 would perfectly fine with me :)

File: src/monitor.js (hunk 1 above)
Comment (imbstack) [inline src/monitor.js:39]: I kinda like having it there for clarity's sake.

Comment (jonasfj) [inline src/monitor.js:39]: okay, sentry will always be a bit of mess... so that's r+ with me...

**Pull request merged**